server/src/session.ts

```diff
@@ -830,6 +834,12 @@ export class Session {
     } else {
       // Immediately enable Legacy/ViewEngine language service
       this.info(`Enabling VE language service for ${projectName}.`);
+      if (!this.ivy && this.angularCoreVersion.major >= 9) {
+        this.connection.sendNotification(SuggestIvyLanguageService, {
+          message: 'An old version of the Angular Language Service is enabled.' +
```

Comment:
nit: `enabled` -> `detected`?
Or just omit the first sentence, and ask users 

> Would you like to enable the new Ivy-native language service to get the latest features and bug fixes?

We can link to the blog post once it's out.